File: Easy/AnagramChecks/AnagramChecks.cpp
bool solve(string s0, string s1) {
    if(s0.length() != s1.length())
        return false;

    sort(s0.begin(), s0.end());
    sort(s1.begin(), s1.end());

    for(int i = 0; i < s0.length(); i++){
        if(s0[i] != s1[i])
            return false;
    }

    return true;
}

File: Easy/AnagramChecks.js
class Solution {
    solve(s0, s1) {
          // sort the strings
    var sortedStr1 = s0.split('').sort().join('');
    var sortedStr2 = s1.split('').sort().join('');
    // compare the sorted strings
    if (sortedStr1 === sortedStr2) {
        return true;
    }
    return false;
    }
}